Formula/questdb.rb

```diff
@@ -0,0 +1,64 @@
+class Questdb < Formula
+  desc "Time Series Database"
+  homepage "https://www.questdb.org"
+  url "https://www.questdb.org/download/questdb-1.0.0-SNAPSHOT-20161013-1304-bin.tar.gz"
```

Comment:
This doesn't look like a stable release: https://github.com/bluestreak01/questdb/releases. We need to use one of those.